### PR TITLE
IVU Feedforward GUI

### PR DESCRIPTION
**Pull request by RafaelLyra8**
Adapt IVU Feedforward GUI to show the IVU parameters
Screenshot from 2024-11-01 18-29-37

### CODE REVIEW

**Review by pnallin — APPROVED** [review]

File: pyqt-apps/siriushla/si_ap_idff/main.py [hunk 1]
```diff
@@ -24,6 +27,12 @@
 from .util import get_idff_icon
 
 
+class SiriusWaveformTable(PyDMWaveformTable):
```
[the remark this answers is not in the document]
Comment (RafaelLyra8) [reply pyqt-apps/siriushla/si_ap_idff/main.py:30]: The tables will be removed and will be replaced by a chart.

**Comment by pnallin** [comment]
> Adapt DELTA Feedforward GUI to show the IVU parameters image


@RafaelLyra8, the waveform PV represents the concatenation of all the 4 individual waveforms (correctors' currents). Should we split the plot into 4, one for each corrector?

**Comment by xresende** [comment]
> > Adapt DELTA Feedforward GUI to show the IVU parameters image
> 
> @RafaelLyra8, the waveform PV represents the concatenation of all the 4 individual waveforms (correctors' currents). Should we split the plot into 4, one for each corrector?

I think it should be one plot for all 4 cors, with the option to select which ones are being rendered.

**Comment by xresende** [comment]
also, white background for the plot is out standard

**Review by anacso17 — APPROVED** [review]

**Pull request merged**